cfme/utils/timeutil.py

```diff
@@ -203,25 +230,47 @@ def to_long_date_format(self):
         return self.strftime(self.long_date_format)
 
     @classmethod
-    def from_saved_report_title_format(cls, time_string):
+    def from_saved_report_title_format(cls, time_string, area="UTC"):
         """ Convert the string representation of the time into parsetime()
 
                 Format here is '%a, %d %b %Y %H:%M:%S +0000'.
 
                 Args:
                     time_string: String with time to parse
+                    area: String indicating a city obtained from
+                        appliance.rest_api.settings["display"]["timezone"]
+
                 Returns: :py:class`utils.timeutil.datetime()` object
-                """
-        return cls._parse(cls.saved_report_title_format, time_string)
+        """
+        return cls._parse(
+            f"{cls.saved_report_title_format} {get_time_difference(area)}", time_string
```

Comment:
If `area` not found then it will formate "something None(area)". I think better to avoid this.